estuary/utils/story.py

```diff
@@ -173,18 +175,24 @@ def get_wait_times(self, results):
             'ContainerKojiBuild': 'completion_time'
         }
 
+        total_wait_time = 0
         wait_times = [None for i in range(len_story - 1)]
 
         for index in range(len_story - 1):
             artifact = results[index]
             next_artifact = results[index + 1]
             property_name = completion_times[artifact.__label__]
             completion_time = getattr(artifact, property_name)
-            if not completion_time or not next_artifact.timeline_timestamp:
+            if not completion_time or not next_artifact.timeline_datetime:
                 continue
 
-            next_artifact_start_time = datetime.strptime(next_artifact.timeline_timestamp,
-                                                         '%Y-%m-%dT%H:%M:%SZ')
+            if next_artifact.__label__.endswith('Advisory'):
+                if next_artifact.attached_build_time(next_artifact, artifact):
```

Comment:
If this evaluates to `False`, then `next_artifact_start_time` will be `None`. An exception will be raised when `next_artifact_start_time = next_artifact_start_time.replace(tzinfo=None)` is executed.

Reply:
What do you think we should put in the else clause? Should we just continue to the next iteration of the loop?

Reply:
That works for me.